src/chat-handler.ts

```diff
@@ -87,7 +104,9 @@ export class ChatHandler extends ChatModel {
       })
     );
 
-    this.updateWriters([{ username: 'AI' }]);
+    const avatar = `data:image/svg+xml;base64,${AI_AVATAR_BASE64}`;
```

Comment:
Maybe `jupyter-chat` could allow taking a `LabIcon` as `avatar_url`, or expose an `avatar` field which could be a `LabIcon`.

Reply:
This interface is a copy of [`User.IIdentity`](https://github.com/jupyterlab/jupyterlab/blob/2f674b04bf33c46da561aa5b74dbc92aed2777c7/packages/services/src/user/index.ts#L235), that is compatible with the [server version of it](https://github.com/jupyter-server/jupyter_server/blob/641e8fccad2e96ba305a74ce83e9420083b7fca6/jupyter_server/auth/identity.py#L35).

In `jupyterlab-chat` extension, the user data are saved in the collaborative document (chat file) in plain text. I don't know if there would be a way to keep the `LabIcon` in this case (some kind of serialization ?)

Reply:
Right, not sure either for now (haven't investigated more), but sticking to this approach is likely fine for now.